doc/en/installation/common/install_packages.rst

```diff
@@ -178,6 +178,8 @@ To make services start automatically during system bootup, run these commands on
     on the database server: ::
     
         # systemctl enable mysql
+    or
+        # systemctl enable mysqld
```

Comment:
That is really `mysql` without the "d":
```
[root@centreon ~]# systemctl enable mysqld
Failed to execute operation: No such file or directory
[root@centreon ~]# systemctl enable mysql
mysql.service is not a native service, redirecting to /sbin/chkconfig.
Executing /sbin/chkconfig mysql on
[root@centreon ~]# rpm -aq |grep Maria
MariaDB-client-10.2.26-1.el7.centos.x86_64
MariaDB-server-10.2.26-1.el7.centos.x86_64
MariaDB-compat-10.2.26-1.el7.centos.x86_64
MariaDB-common-10.2.26-1.el7.centos.x86_64
```

Reply:
forget this.. now that i saw you put an "or" ... sorry

Reply:
indeed with mariaDb, the daemon is linked to one named mysql.service.
But on mysql 8, the daemon is named mysqld :
`
[root@localhost ~]# systemctl | grep sql
  mysqld.service                                
[root@localhost ~]# rpm -qa | grep ysq
mysql80-community-release-el7-3.noarch
mysql-community-common-8.0.17-1.el7.x86_64
mysql-community-client-8.0.17-1.el7.x86_64
mysql-community-server-8.0.17-1.el7.x86_64
mysql-community-libs-8.0.17-1.el7.x86_64
mysql-community-libs-compat-8.0.17-1.el7.x86_64
[root@localhost ~]# 
`